projects/python/perception/object_detection_2d/yolov5/inference_tutorial.ipynb

```diff
@@ -0,0 +1,305 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import torch\n",
+    "import cv2\n",
+    "%matplotlib inline\n",
+    "from matplotlib import pyplot as plt\n",
+    "from opendr.engine.data import Image\n",
+    "from opendr.perception.object_detection_2d import draw_bounding_boxes\n",
+    "\n",
+    "for f in 'zidane.jpg', 'bus.jpg':\n",
+    "    torch.hub.download_url_to_file('https://ultralytics.com/images/' + f, f)  # download 2 images\n",
+    "\n",
+    "\n",
+    "im1 = Image.open('zidane.jpg')\n",
+    "results = learner.infer(im1)\n",
+    "im1_dets = draw_bounding_boxes(im1.opencv(), results, learner.classes, show=False)\n",
```

Comment:
Similar to the earlier comment, i think that it would be helpful to add `line_thickness` to clearly show that it can be changed. I remember this being an issue for use case partners when they provided feedback to similar detectors.

```suggestion
    "im1_dets = draw_bounding_boxes(im1.opencv(), results, learner.classes, show=False, line_thickness=3)\n",
```